README.md

```diff
@@ -1,180 +1,75 @@
-# Swift-DocC
-
+<h1 align="center">
+    Swift-DocC
+    <br>
+    <br>
+    <a href="https://www.swift.org/documentation/docc/">
+        <img src="https://www.swift.org/images/docc-hero@2x.png" alt="DocC" width="500">
+    </a>
+    <br>
+</h1>
+
+<h4 align="center">
 Swift-DocC is a documentation compiler for Swift frameworks and packages aimed 
 at making it easy to write and publish great developer documentation.
+</h4>
 
 For an example of Swift-DocC in action, check out 
 [developer.apple.com](https://developer.apple.com/documentation).
 Much of Apple's developer documentation,
 from [Reference documentation](https://developer.apple.com/documentation/GroupActivities)
 to [Tutorials](https://developer.apple.com/tutorials/swiftui),
+and [Long-form Content](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/)
 is built using Swift-DocC.
 
+To learn more about the essentials of this tool 
+refer to the
+[user documentation](https://www.swift.org/documentation/docc).
+
 Swift-DocC is being actively developed. For more information about the
 Swift-DocC project, see the introductory blog post
 [here](https://swift.org/blog/swift-docc/).
 
```

Comment:
Perhaps to separate the concerns further, link to contributing like that? That's a nitpick, feel free to discard, non-blocking. 

```suggestion
To learn more about how Swift-DocC works internally, and how to contribute to it, please see [CONTRIBUTING.md](CONTRIBUTING.md)
```